.claude/rules/git-push-safety.md

```
@@ -0,0 +1,36 @@
+## 공유 브랜치 보호 & push 안전
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)